rustls/src/msgs/handshake_test.rs

```diff
@@ -813,14 +814,14 @@ fn get_sample_certificatepayloadtls13() -> CertificatePayloadTls13 {
 }
 
 fn get_sample_serverkeyexchangepayload_ecdhe() -> ServerKeyExchangePayload {
```

Comment:
I think we also need to add a similar `fn get_sample_serverkeyexchangepayload_ffdhe`.
And add it into `fn get_all_tls12_handshake_payloads` below

Reply:
Will do!